### PR TITLE
Remove pass test

### PR DESCRIPTION
**Pull request by mubarak23**
Remove pass test PR
tests/ui/cfg/cfg-panic-abort.rs
tests/ui/fmt/format-args-capture.rs
tests/ui/panic-runtime/link-to-abort.rs
tests/ui/panic-runtime/abort-link-to-unwinding-crates.rs
tests/ui/panic-runtime/abort.rs
tests/ui/statics/issue-91050-1.rs
tests/ui/statics/issue-91050-2.rs
tests/ui/debuginfo/debuginfo-emit-llvm-ir-and-split-debuginfo.rs
tests/ui/panic-runtime/lto-abort.rs

### CODE REVIEW

**Comment by mubarak23** [comment]
Sorry, i close the other PR by mistake

**Comment by antoyo** [comment]
Some tests are failing, so please add them back to the file.

**Comment by mubarak23** [comment]
I will look for them and add them back

**Comment by mubarak23** [comment]
add the following back to the list
- format-args-capture.rs
- tests/ui/cfg/cfg-panic-abort.rs

File: tests/failing-ui-tests.txt [hunk 1]
```diff
@@ -16,11 +15,11 @@ tests/ui/sepcomp/sepcomp-statics.rs
 tests/ui/asm/x86_64/may_unwind.rs
 tests/ui/backtrace.rs
 tests/ui/catch-unwind-bang.rs
-tests/ui/cfg/cfg-panic-abort.rs
 tests/ui/drop/dynamic-drop-async.rs
+tests/ui/cfg/cfg-panic-abort.rs
 tests/ui/drop/repeat-drop.rs
-tests/ui/fmt/format-args-capture.rs
 tests/ui/coroutine/panic-drops-resume.rs
+format-args-capture.rs
```
Comment (antoyo) [inline tests/failing-ui-tests.txt]: It should be:
```suggestion
tests/ui/fmt/format-args-capture.rs
```

**Comment by antoyo** [comment]
There are still a few failures, as you can see [here](https://github.com/antoyo/rustc_codegen_gcc/actions/runs/8392175674/job/22984213191?pr=5#step:13:4502).
Please add back these failures.

**Comment by mubarak23** [comment]
Add the following back to the list

- tests/ui/panic-runtime/abort-link-to-unwinding-crates.rs
- tests/ui/panic-runtime/abort.rs
- tests/ui/panic-runtime/lto-abort.rs

File: tests/failing-ui-tests.txt [hunk 2]
```diff
@@ -30,7 +29,7 @@ tests/ui/mir/mir_drop_order.rs
 tests/ui/mir/mir_let_chains_drop_order.rs
 tests/ui/oom_unwind.rs
 tests/ui/panic-runtime/abort-link-to-unwinding-crates.rs
-tests/ui/panic-runtime/abort.rs
+tests/ui/panic-runtime/abort.rs
```
Comment (antoyo) [inline tests/failing-ui-tests.txt]: Please remove the trailing space.

**Comment by antoyo** [comment]
From what I can see in the [logs](https://github.com/antoyo/rustc_codegen_gcc/actions/runs/8403297170/job/23013650782?pr=5#step:15:1585), it seems like these tests pass and should be removed:

* tests/ui/binding/fn-arg-incomplete-pattern-drop-order.rs
* tests/ui/macros/rfc-2011-nicer-assert-messages/all-expr-kinds.rs
* tests/ui/runtime/rt-explody-panic-payloads.rs

Did you try removing them?

**Comment by mubarak23** [comment]
The list I shared at the start of this PR was the one I removed. I will review the logs and remove any that have passed and I may have missed.

**Comment by antoyo** [comment]
Please add `tests/ui/binding/fn-arg-incomplete-pattern-drop-order.rs` to [`failing-lto-tests.txt`](https://github.com/antoyo/rustc_codegen_gcc/blob/master/tests/failing-lto-tests.txt) as it seems this test only fails with LTO enabled.

**Comment by mubarak23** [comment]
Alright

**Comment by antoyo** [comment]
Thanks for your contribution!

**Review by antoyo — APPROVED** [review]

**Pull request merged**